src/optimizer/optimizer.cpp

```diff
@@ -115,7 +114,7 @@ shared_ptr<planner::AbstractPlan> Optimizer::BuildPelotonPlanTree(
     ExprMap output_expr_map;
     auto best_plan = ChooseBestPlan(root_id, properties, &output_expr_map);
     if (best_plan == nullptr) return nullptr;
-
+    
```

Comment:
I am sorry, because there are too many newlines in peloton and I don't know which is necessary and which is not. After I debug and remove the log statement, sometimes I just left newline